lib/assets/javascripts/twine.js.coffee

```diff
@@ -293,11 +296,12 @@ setupAttributeBinding = (attributeName, bindingName) ->
       newValue = fn.call(node, context, rootContext)
       newValue = !!newValue if booleanAttribute
       return if newValue == lastValue
+      attributeName = 'readOnly' if attributeName == 'readonly'
```

Comment:
This also because `readOnly` is a special attribute on elements (and `readonly` is not equivalent) but the binds should be case-insensitive